.nengobones.yml

```diff
@@ -24,6 +24,8 @@ travis_yml:
   deploy_dists:
     - sdist
     - bdist_wheel
+  bones_install: >-
+    git+https://github.com/nengo/nengo-bones.git@$(git symbolic-ref --short HEAD)
```

Comment:
This goes past the 80 character limit with or without the linebreak, so I'd probably just put it all on one line to avoid the extra continuation character stuff.  Either way though 🤷‍♂️